clippy_lints/src/deprecated_lints.rs

```diff
@@ -192,6 +207,7 @@ declare_deprecated_lint! {
     /// ### Deprecation reason
     /// The `avoid_breaking_exported_api` config option was added, which
     /// enables the `wrong_self_conversion` lint for public items.
+    #[clippy::version = "1.54.0"]
```

Comment:
What is being displayed for deprecated lints? Also "Added in: <version>"? If so, this is probably wrong, since we decided earlier to put the version when the lint was removed here.

Reply:
Good catch! I've updated the page to display "Deprecated in `<version_label>`" for these lints.